### ISSUE TITLE
no buttons for registered user to fill text poll - maybe Uncaught (in promise) TypeError: can't convert null to object - 

### ISSUE THREAD

**Issue by devnull4242** [nextcloud/polls#3325]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### What went wrong, what did you observe?

It is possible to fill the form (text) with anonymous user. But with a registered user there are only shown the old values and there is no line with the form for the actual user. I use local users but it also seems to be a problem with ldap see https://help.nextcloud.com/t/umfragen-mit-ldap-anbindung/181888 . 

Firefox error (maybe belongs to the problem or not)
https://i.imgur.com/zkBN6wQ.png
Firefox screenshot, no button for the Nextcloud registered user "test"
https://i.imgur.com/wq5wOhO.png

### What did you expect, how polls should behave instead?

After login with the user i expect to see the text fields to input the poll.

### What steps does it need to replay this bug?

1. Normal Nextcloud installation Nextcloud 28.0.2
2. Normal Polls version 6.0.1
3. create normal poll

### Installation method

Installed/updated from the appstore (Apps section of your site)

### Installation type

Updated from a much older version (more than one major version)

### Affected polls version

6.0.1

### Which browser did you use, when experiencing the bug?

- [X] Firefox
- [X] Chrome
- [ ] Chromium/Chromium based (i.e. Edge)
- [ ] Safari
- [ ] Other/Don't know

### Add your browser log here

Uncaught (in promise) TypeError: can't convert null to object
    setPreference settings.js:60
    _ vuex.esm.js:844
    commit vuex.esm.js:466
    commit vuex.esm.js:465
    _withCommit vuex.esm.js:624
    commit vuex.esm.js:464
    commit vuex.esm.js:409
    commit vuex.esm.js:796
    get settings.js:113
    _ vuex.esm.js:851
    dispatch vuex.esm.js:516
    dispatch vuex.esm.js:406
    created App.vue:97
    VueJS 17
    <anonymous> main.js:56
    <anonymous> main.js:63
    <anonymous> main.js:63
settings.js:60:9

### Additional client environment information

```
setPreference(state, payload) {
		Object.keys(payload).filter((key) => key in state.user).forEach((key) => {
			state.user[key] = payload[key]
		})
	},
```
[polls/src/js/store/modules/settings.js](https://github.com/nextcloud/polls/blob/master/src/js/store/modules/settings.js)

### NC version

Nextcloud 25 (Nextcloud Hub 3)

### Other Nextcloud version

Nextcloud 28.0.2

### PHP engine version

PHP 8.1

### Database engine

MariaDB

### Which user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other/Don't know

### Add your nextcloud server log here

```shell
no logs at this time
```

### Additional Information

I only noticed the error when I looked at this thread https://help.nextcloud.com/t/umfragen-mit-ldap-anbindung/181888. I don't actually use Nextcloud Polls.

**Comment by dartcafe** [comment]
![grafik](https://github.com/nextcloud/polls/assets/26707476/70a0e0b2-066c-4e28-83c8-2f5207ba5a9f)

this tells me the settings are null (aka not set). This is a known error https://github.com/nextcloud/polls/issues/3283

Please update to the current version and check, if this error still exists. If it does, please post your poll settings and any further error logs (if there are any). 

Notice: I replaced the imgur links in your post with the images.

**Comment by dartcafe** [comment]
I assume, this error is more a duplicate of #3313

**Comment by devnull4242** [comment]
Problem solved with Polls 6.1.1
